airflow/www/static/js/App.tsx

```diff
@@ -55,6 +56,8 @@ interface AppProps extends PropsWithChildren {
   cache: EmotionCache;
 }
 
+const isInputInFocus: IsInputInFocus = "isInputInFocus";
```

Comment:
it's just a string we don't need a special type for it.

```suggestion
const isInputInFocus = "isInputInFocus";
```

Reply:
Here the idea behind having a type for a string was to enforce this to have only specific values (since it is the key of the local storage object). In the above case, it was below.
```
// In type/index.ts
type IsInputInFocus = "isInputInFocus";
```
We can replace it with a global constant that will be imported wherever used. Where can I define a constant to import it anywhere required? I do not seem to find where we define all constants used in UI.

Reply:
Just exporting the string from the useKeysPress is fine.

Reply:
Done. Defined in `useKeysPress` and importing it everywhere.